pop_tools/grid.py

```diff
@@ -132,6 +132,12 @@ def get_grid(grid_name, scrip=False):
     assert kmt_flat.max() <= len(z_t), 'Max KMT > length z_t'
     KMT = kmt_flat.reshape(grid_attrs['lateral_dims']).astype(np.int32)
 
+    # derive KMU
+    KMU = np.zeros_like(KMT)
+    for i in range(KMT.shape[0]):
+        for j in range(KMT.shape[1]):
+            KMU[i, j] = min(KMT[i, j], KMT[i - 1, j], KMT[i, j - 1], KMT[i - 1, j - 1])
```

Comment:
Perhaps there's a vectorized way to do this? I was thinking it could be sped up through list comprehension. But it's quite quick for the coarse grids. My concern is the 1/10th degree grids.